packages/app/src/cli/models/app/loader.ts

```diff
@@ -468,9 +400,7 @@ async function logMetadataForLoadedApp(
   app: App,
   loadingStrategy: {
     usedCustomLayoutForWeb: boolean
-    usedCustomLayoutForUIExtensions: boolean
-    usedCustomLayoutForFunctionExtensions: boolean
-    usedCustomLayoutForThemeExtensions: boolean
+    usedCustomLayoutForExtensions: boolean
   },
 ) {
   await metadata.addPublicMetadata(async () => {
```

Comment:
Do you think there's any additional metadata it might be worth including, in light of this new approach?

Reply:
Not that I kind think off, but we are still tracking the 3 categories, we should think of a way to track any kind of extension 🤔 Can we track extension types in an array? what would be the best format for that?